openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -79,6 +84,9 @@ The following items are out of scope for the current version of this document, b
 
 * Trust Management, i.e. authorization of an issuer to issue certain types of credentials, authorization of the Wallet to be issued certain types of credentials, authorization of  the Verifier to receive certain types of credentials.
 * Protocol for presentation of Verifiable Credentials for offline use-cases, e.g. over BLE.
+* Profile of OID4VCI to issue ISO mdocs is defined in [@!ISO.23220-3].
```

Comment:
If we're not going to add a reference to this (due to it showing as deleted on the ISO website) we'll need to make this not a reference for the compile to succeed:

```suggestion
* Profile of OID4VCI to issue ISO mdocs is defined in ISO 23220-3.
```